### DISCUSSION TITLE
New address of API?

### DISCUSSION THREAD

**Discussion by sasbibic** [bilan/connectlife-api-connector#28] [General]
I'm was accessing  api.connectlife.io with small nodejs server, using Your project as example for authentication to connectlife. Now this is obviously not possible anymore. Authentication still works. I can see in Your code that You are using new API endpoint connectlife.bapi.ovh/appliances. But when I'm trying to access this address I get 403 error. Yes I've specified X-Token header, but simply accessing address from browser I get web page telling me that I'm blocked (Sorry, you have been blocked You are unable to access bapi.ovh by Cloudflare security). Is this address somehow limited to specific region ?

Regards
/Sas

**Comment by trajano** [comment]
See https://github.com/oyvindwe/connectlife-ha/issues/6#issuecomment-2207510322 you need the `User-Agent` value.
